src/zui/ZUISwitch/index.stories.tsx

```diff
@@ -20,33 +20,39 @@ export const Default: Story = {
     return (
       <ZUISwitch
         checked={checked}
-        onChange={() => setChecked(!checked)}
+        disabled={args.disabled}
+        label={args.label}
+        onChange={(newCheckedState) => setChecked(newCheckedState)}
```

Comment:
Not all properties are being applied here, so `labelPlacement` is not working in Storybook. Maybe pass `{...args}` instead?

Reply:
thank you for noticing this! :D